worker.go

```diff
@@ -462,6 +466,38 @@ func CollectResult(data []Stats, mode bool, chars []string, detailed bool) (resu
 		result.DPSByTarget[j] = dpsTargetRollup
 	}
 
+	// required ER
+
+	if erCalcMode {
```

Comment:
So I was suggesting for this part, we do something like this:

```go
if erCalcMode {
    energyCount := make([][]float64, charCount)

    for i := 0; i < charCount; i++ {
        energyCount[i] = make([]float64, charMaxEnergy) //need to prob add a database for this or soemthing so we know what the max is?
    }

    for _, d := range data {
        for j, charEnergy := range d.EnergyWhenBurst {
            for _, v := range charEnergy {
                energyCount[j][v]++
            } 

        }
    }

    //so now you have basically distribution of various energy and their frequency. I would use this then to calculate
    //the min/max/std dev of raw energy received

}
```

benefit of this approach is now we are able to summarize what's the min/mean/max/std. dev. of raw energy received. it gives us a better picture of how volatile the energy generation of this team is

and then we can calculate the min and average ER threshhold.

Reply:
Resolved, additional representation of the data will be added later on